ocaml/xapi/xapi_vm_lifecycle.ml

```diff
@@ -728,6 +741,14 @@ let assert_operation_valid ~__context ~self ~op ~strict =
   | Some (a, b) ->
       raise (Api_errors.Server_error (a, b))
 
+(* can't put this into xapi_vtpm because it creates a cycle *)
```

Comment:
OK now for such a simple function. If this gets more complex, then perhaps put it in a separate file like xapi_vtpm_helpers.ml.